just.py

```diff
@@ -41,25 +42,35 @@ def parseTasks(args):
     task_id = None
     task_name = None
     task_body = []
+    task_parents = dict()
     for line in lines:
         m_start = re.match(r"(.*):(.*):.*{{", line)
         m_end = re.match(r"^#*}}\s*$", line)
+        m_parent = re.match(r"\s*PARENTS:", line)
 
         if m_end:
             is_in = False
             task_id = int(task_id)
+            if task_id > 0 and task_id not in task_parents:
+               task_parents[task_id] = [task_id - 1] # Default case, just wait for prev to finish
             tasks[task_id] = {'task_name': task_name, 'task_body': task_body}
             task_body = []
 
-        if is_in:
+        if m_parent:
+            line = line.lstrip("PARENTS:")
+            pattern = re.compile(r'\s+')
+            line = re.sub(pattern, '', line)
```

Comment:
This will even delete internal spaces; use `parents = [part.strip() for part in line.split(',')]` instead